tslint.json

```diff
@@ -11,6 +11,7 @@
         "indent": [true,
             "spaces"
         ],
+        "inline-comment-spacing": true,

```

Comment:
Are `scripts/tslint` rules accessible outside our repo? Are rules in this directory a sort of test-bed for adding rules to tslint itself?

If these are only used by us, then that works for me. We probably shouldn't rename these if someone else is using the `boolean-trivia` rule.